src/main/java/com/databricks/hadoop/mapreduce/lib/input/XmlInputFormat.java

```diff
@@ -29,10 +30,11 @@
 import org.apache.hadoop.mapreduce.InputSplit;
 import org.apache.hadoop.mapreduce.RecordReader;
 import org.apache.hadoop.mapreduce.TaskAttemptContext;
+import org.apache.hadoop.mapreduce.lib.input.FileSplit;
+import org.apache.hadoop.mapreduce.lib.input.TextInputFormat;
```

Comment:
In alphabetic ordering, this `org.apache.hadoop.mapreduce.lib` should go above but I will update them when downgrading the API version in another PR for Hadoop version compatibility, which I will open right after closing this.